docs/en/source/opcua-IoTIntegration.rst

```diff
@@ -0,0 +1,627 @@
+.. figure:: figures/OPC-UA.png
+
+
+Integrate Scorpio Broker with OPC UA IoT agent 
+****************************************************
+
+This is a step-by-step tutorial that will introduce in detail how to enable OPC UA to FIWARE connecting an OPC UA server to Context Broker using the agent. The OPC UA data will be automatically published in a FIWARE Context Broker using NGSI data model.
+
+**What is OPC UA?**
+
+OPC UA is a well-known client-server protocol used in the Industry.
+
+In real environments, an OPC UA server is usually responsible for fetching sensor data from factory-level machinery making them available to an OPC UA client (the Agent in our case).
```

Comment:
Write "IoT agent in our case" instead of "the agent in our case"

Reply:
Done